yarn.lock

```diff
@@ -9451,6 +9451,14 @@ enhanced-resolve@^4.3.0:
     memory-fs "^0.5.0"
     tapable "^1.0.0"
 
+enhanced-resolve@^5.8.3:
```

Comment:
Do you know why this file was changed? :thinking:

Reply:
Sorry, I have no idea... 🤷🏼 
I don't even have this line in my local repo